src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/CreateCourtBookingService.kt

```diff
@@ -60,6 +60,8 @@ class CreateCourtBookingService(
       comments = request.comments,
       videoUrl = request.videoLinkUrl,
       createdBy = createdBy,
+      notesForStaff = request.notesForStaff?.takeIf { createdBy is ExternalUser },
```

Comment:
We will only use private notes when requested by an external user.

Reply:
ditto.